seeds/zeroex/optimism/zeroex_api_optimism_fills_sample.csv

```diff
@@ -0,0 +1,6 @@
+tx_hash,block_number,evt_index,contract_address,block_time,maker,taker,taker_token,taker_symbol,maker_token,maker_symbol,token_pair,taker_token_amount,taker_token_amount_raw,maker_token_amount,maker_token_amount_raw,type,affiliate_address,swap_flag,matcha_limit_order_flag,volume_usd,tx_from,tx_to
+0xaebe0d75b0518a40f164d200a8a4705300c7bb9a389b3dd909c0c9c350ac5aba,48984245,22,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-12-13 13:30,0x00000012556e6973776170563300000000000000,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,DAI,0x4200000000000000000000000000000000000006,WETH,DAI-WETH,575.6690579,5.76E+20,0.4378764528,4.38E+17,BridgeFill,0x1000000000000000000000000000000000000011,TRUE,FALSE,566.0560268,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0x0575191a55d6f28bee4963fcc76a7c6c5428546c
+0xaebe0d75b0518a40f164d200a8a4705300c7bb9a389b3dd909c0c9c350ac5aba,48984245,22,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-12-13 13:30,0x00000012556e6973776170563300000000000000,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,DAI,0x4200000000000000000000000000000000000006,WETH,DAI-WETH,575.6690579,5.76E+20,0.4378764528,4.38E+17,BridgeFill,0x1000000000000000000000000000000000000011,TRUE,FALSE,566.0560268,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0x0575191a55d6f28bee4963fcc76a7c6c5428546c
+0xb83179df3e200111311ccc21c61367d56f6773a10abd77a4168c6ef92606b0ba,48657877,10,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-12-13 2:59,0x0000001942656574686f76656e78000000000000,0x1e5d675a0204cc2d55a5c87f1de06c37eef74337,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,DAI,0x4200000000000000000000000000000000000006,WETH,DAI-WETH,20.78843642,2.08E+19,0.01636720389,1.64E+16,BridgeFill,0x86003b044f70dac0abc80ac8957305b6370893ed,TRUE,FALSE,20.78094777,0x1e5d675a0204cc2d55a5c87f1de06c37eef74337,0xdef1abe32c034e558cdd535791643c58a13acc10
+0x8f651b59f4b4252d4fb70900dca532dfd6683e728a8b7375c0a66d0aa3103f1e,48705440,6,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-12-13 4:49,0x00000012556e6973776170563300000000000000,0x88b9ba802462c095305c50937f46949e7e4517c5,0x4200000000000000000000000000000000000042,OP,0x7f5c764cbc14f9669b88837ca1490cca17c31607,USDC,OP-USDC,20,2.00E+19,20.617021,2.06E+07,BridgeFill,0x720c9244473dfc596547c1f7b6261c7112a3dad4,TRUE,FALSE,20.62196909,0x88b9ba802462c095305c50937f46949e7e4517c5,0xdef1abe32c034e558cdd535791643c58a13acc10
+0xaebe0d75b0518a40f164d200a8a4705300c7bb9a389b3dd909c0c9c350ac5aba,48984245,15,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-12-13 13:30,0x00000012556e6973776170563300000000000000,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,DAI,0x4200000000000000000000000000000000000006,WETH,DAI-WETH,1151.338116,1.15E+21,0.8717674699,8.72E+17,BridgeFill,0x1000000000000000000000000000000000000011,TRUE,FALSE,1126.959961,0xe075e62d1b8dc71c14eda5298327ae19cc6a7996,0x0575191a55d6f28bee4963fcc76a7c6c5428546c
```

Comment:
Hi @RantumBits, please have a look on [this query](https://dune.com/queries/1756506?d=1).

It seems these seeds do not appear in the model results.

test_description | result (model) | expected (seed) | tx_hash | evt_index
-- | -- | -- | -- | --
matched records count | 0 | 1 | 0xb83179df3e200111311ccc21c61367d56f6773a10abd77a4168c6ef92606b0ba | 10
matched records count | 0 | 1 | 0xaebe0d75b0518a40f164d200a8a4705300c7bb9a389b3dd909c0c9c350ac5aba | 15
matched records count | 0 | 1 | 0x8f651b59f4b4252d4fb70900dca532dfd6683e728a8b7375c0a66d0aa3103f1e | 6
matched records count | 0 | 1 | 0xaebe0d75b0518a40f164d200a8a4705300c7bb9a389b3dd909c0c9c350ac5aba | 22